README.md

```diff
@@ -1,57 +1,108 @@
-- !pause !resume and adjust the !volume
-
-And more! Try !help to get a full list of available commands
+Invite the bot to your server: [here](https://discordapp.com/oauth2/authorize?client_id=345207331295854593&scope=bot&permissions=66186303).
+
+# Modules
+
+Kirbi absorbs the functionality of whatever you add to him! We've included a few base features to help with your plugins, which we will describe later (probably in a wiki or something).
+
+Modules for Kirbi can add commands or other functionality. For example, the [kirbi-beer-lookup](https://github.com/Richardson-Media-House/kirbi-beer-lookup) module adds the !brew command which returns information on breweries and specific brews!
+
+## Using Modules
+
+Modules can be autoloaded from either a single file in `./modules/module-name.js` or an NPM module/Github repo named `kirbi-module-name`.
+
+To autoload a module, add the plugin name to the `externalModules` (or the `modules` array under the appropriate api name for an api-specific module) array in `config/config.json` (without the `kirbi-`):
+
+```json
+{
+	...
+	"externalModules": ["beer-lookup", "xkcd", "urbandictionary", "wikipedia", "cocktail-lookup", "dice", "dictionary", "misc", "random", "translator", "rss"],
+	"elizaEnabled": true,
+	"discord": {
+		"enabled": true,
+		"serverName": "insert your server name",
+		"welcomeChannel": "insert channel id of channel where you want welcome to be",
+		"modules": ["musicplayer", "moderation"],
+		...
+	}
+}
+```
+
+and make sure to include any external modules as dependencies in the `package.json` file:
+
+```json
+{
+  ...
+	"dependencies": {
+		"chalk": "^2.0.1",
+		"kirbi-beer-lookup": "Richardson-Media-House/kirbi-beer-lookup",
+		"kirbi-cocktail-lookup": "Richardson-Media-House/kirbi-cocktail-lookup",
+		"kirbi-dice": "Richardson-Media-House/kirbi-dice",
+		"kirbi-dictionary": "Richardson-Media-House/kirbi-dictionary",
+		"kirbi-discord": "Richardson-Media-House/kirbi-discord",
+		"kirbi-discord-moderation": "Richardson-Media-House/kirbi-discord-moderation",
+		"kirbi-discord-musicplayer": "Richardson-Media-House/kirbi-discord-musicplayer",
+		"kirbi-misc": "Richardson-Media-House/kirbi-misc",
+		"kirbi-urbandictionary": "Richardson-Media-House/kirbi-urbandictionary",
+		"kirbi-random": "Richardson-Media-House/kirbi-random",
+		"kirbi-rss": "Richardson-Media-House/kirbi-rss",
+		"kirbi-translator": "Richardson-Media-House/kirbi-translator",
+		"kirbi-wikipedia": "Richardson-Media-House/kirbi-wikipedia",
+		"kirbi-xkcd": "Richardson-Media-House/kirbi-xkcd"
+ 	},
+  ...
+}
+```
+
+
+## Included Modules
+List of available modules (via `./modules/module-name`):
+
+- `admin` => adds a few administrative commands for server owners and bot admins to use.
+- `server` => lists out a given list of servers and information about them.
+- `welcome` => spits out a given markdown file in a pretty embed format. Useful for making a welcome/rules channel look better than if you typed it yourself.
```

Comment:
We can, but it might be something we want to break out later. In the meantime we should note which one requires the bot to be re-hosted to be used, that way users will know they can't use them just by inviting the bot.